docs/main.tex

```diff
@@ -1,117 +1,25 @@
-\documentclass[graybox,envcountchap,sectrefs]{scn/svmono}
+\documentclass{scn/scndocument}
```

Comment:
Latex unable to resolve relative paths, so we must use `\documentclass{scndocument}` and add `scn` folder via **TEXINPUTS** environment variable